index.js

```diff
@@ -28,286 +30,296 @@ function isMusl() {
+            localFileExisted = existsSync(
+              join(__dirname, 'x-win.linux-riscv64-musl.node')
+            )
+            try {
+              if (localFileExisted) {
+                nativeBinding = require('./x-win.linux-riscv64-musl.node')
+              } else {
+                nativeBinding = require('@miniben90/x-win-linux-riscv64-musl')
+              }
+            } catch (e) {
+              loadError = e
+            }
+          } else {
+            localFileExisted = existsSync(
+              join(__dirname, 'x-win.linux-riscv64-gnu.node')
+            )
+            try {
+              if (localFileExisted) {
+                nativeBinding = require('./x-win.linux-riscv64-gnu.node')
+              } else {
+                nativeBinding = require('@miniben90/x-win-linux-riscv64-gnu')
+              }
+            } catch (e) {
+              loadError = e
+            }
+          }
+          break
+        case 's390x':
           localFileExisted = existsSync(
-            join(__dirname, 'x-win.linux-riscv64-gnu.node')
+            join(__dirname, 'x-win.linux-s390x-gnu.node')
           )
           try {
             if (localFileExisted) {
-              nativeBinding = require('./x-win.linux-riscv64-gnu.node')
+              nativeBinding = require('./x-win.linux-s390x-gnu.node')
             } else {
-              nativeBinding = require('@miniben90/x-win-linux-riscv64-gnu')
+              nativeBinding = require('@miniben90/x-win-linux-s390x-gnu')
             }
           } catch (e) {
             loadError = e
           }
-        }
-        break
-      case 's390x':
-        localFileExisted = existsSync(
-          join(__dirname, 'x-win.linux-s390x-gnu.node')
-        )
-        try {
-          if (localFileExisted) {
-            nativeBinding = require('./x-win.linux-s390x-gnu.node')
-          } else {
-            nativeBinding = require('@miniben90/x-win-linux-s390x-gnu')
-          }
-        } catch (e) {
-          loadError = e
-        }
-        break
-      default:
-        throw new Error(`Unsupported architecture on Linux: ${arch}`)
+          break
+        default:
+          throw new Error(`Unsupported architecture on Linux: ${arch}`)
+      }
+      break
+    default:
+      throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`)
+  }
+  
+  if (!nativeBinding) {
+    if (loadError) {
+      throw loadError
     }
-    break
-  default:
-    throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`)
-}
-
-if (!nativeBinding) {
-  if (loadError) {
-    throw loadError
+    throw new Error(`Failed to load native binding`)
   }
-  throw new Error(`Failed to load native binding`)
 }
 
 const { WindowInfo, activeWindow, activeWindowAsync, openWindows, openWindowsAsync, subscribeActiveWindow, unsubscribeActiveWindow, unsubscribeAllActiveWindow, installExtension, uninstallExtension, enableExtension, disableExtension } = nativeBinding
```

Comment:
If it's use without calling init it will never work.

Reply:
Yeah, the idea is that init should be called outside of the library.